src/utils/fetchLogFile/index.ts

```diff
@@ -1,7 +1,15 @@
+enum IncompleteDownloadReason {
+  SERVER_ERROR = "SERVER_ERROR",
+  FILE_TOO_LARGE = "FILE_TOO_LARGE",
```

Comment:
[nit] Standard uppercase would be more in line with enums across Parsley
```suggestion
  ServerError = "SERVER_ERROR",
  FileTooLarge = "FILE_TOO_LARGE",
```